### DISCUSSION TITLE
其他的大佬有写邮件的习惯吗？

### DISCUSSION THREAD

**Discussion by Luoyingliang** [avaloniachina/Documentation#9] [Q&A] [answered]
其他的大佬有写邮件的习惯吗？有必要保持最低限度的沟通不？

**Comment by rabbitism** [comment]
我觉得你说的很有道理……我甚至都没有注意到这里有人发起了讨论

**Reply by Luoyingliang** [reply]
其实我看到邮件距离收到已经过去了6个小时了，因为最近需要用Avalonia重写项目，就想着把文档完善一下

**Reply by Luoyingliang** [reply]
其实如果可以，可以公开一些联系方式的，有时候提pr之后也没人响应~

**Reply by rabbitism** [reply]
> 其实如果可以，可以公开一些联系方式的，有时候提pr之后也没人响应~

其实是有个微信群的，只是不知道你愿不愿意加入。

**Comment by Luoyingliang** [comment]
可以的！&nbsp;


Acoris
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Dong ***@***.***&gt;; 
发送时间: 2022年12月15日(星期四) 凌晨0:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [avaloniachina/Documentation] 其他的大佬有写邮件的习惯吗？ (Discussion #9)





  
其实如果可以，可以公开一些联系方式的，有时候提pr之后也没人响应~
  
其实是有个微信群的，只是不知道你愿不愿意加入。

**Comment by rabbitism** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/14807942/207782822-df10c6e1-1a31-4e43-add6-a37cb771cb30.png)
